pkg/features/kube_features.go

```diff
@@ -1112,7 +1112,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	SizeMemoryBackedVolumes: {Default: true, PreRelease: featuregate.Beta},
 
-	StableLoadBalancerNodeSet: {Default: true, PreRelease: featuregate.Beta},
+	StableLoadBalancerNodeSet: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.30, remove in 1.31
```

Comment:
You have to update line 750 too

Reply:
Isn't that indicating when the feature gate was promoted to beta?

Reply:
It's the record of which releases had which stability

Reply:
OK, updated the PR and added a line indicating that GA is starting 1.30